apps/website/src/components/calendar/Calendar.tsx

```diff
@@ -383,20 +412,14 @@ const Calendar = ({
                       <p
                         className={classes(
                           "flex gap-1 rounded-bl-lg px-1.5 pb-1 pt-1.5 font-mono text-sm leading-none md:-mr-1 md:-mt-1",
-                          // Fade out days in the past
-                          fullDate.getTime() < today.getTime() && "opacity-50",
-                          // Show the current day in a pill
-                          fullDate.getTime() === today.getTime() &&
-                            theme.heading,
+                          isPast && "opacity-50",
+                          isToday && theme.heading,
                         )}
                       >
-                        {date.toLocaleString(undefined, {
-                          minimumIntegerDigits: 2,
-                        })}
-
+                        {paddedDate}
                         {/* Render the day of the week for mobile */}
                         <span className="md:hidden">
-                          {days[fullDate.getDay()]?.slice(0, 1)}
+                          {days[(i + startDay) % 7]?.charAt(0)}
```

Comment:
You seem to be off by one here?

Reply:
I think it works now?  https://github.com/alveusgg/alveusgg/pull/744/commits/a15d770e22746bb3e48fa2ddbe960b747b039271 
The date used for selected month/year was not starting on the first day of the month, it was the current date instead hence incorrect weekday value.